app/jobs/reports/monthly_account_reuse_report.rb

```diff
@@ -0,0 +1,157 @@
+require 'csv'
+
+module Reports
+  class MonthlyAccountReuseReport < BaseReport
+    REPORT_NAME = 'monthly-account-reuse-report'
+
+    attr_reader :report_date
+
+    def perform(report_date)
+      @report_date = report_date
+
+      _latest, path = generate_s3_paths(REPORT_NAME, 'json', now: report_date)
+      body = report_body.to_json
+
+      [
+        bucket_name, # default reporting bucket
+        IdentityConfig.store.s3_public_reports_enabled && public_bucket_name,
+      ].select(&:present?).
```

Comment:
we're not putting these in the public reporting bucket so we can remove the loop on this and just use `bucket_name`

Reply:
Continued in another thread